public/static/docs/install/index.md

```diff
@@ -7,6 +7,10 @@ Please double check that you don't already have DVC (for example running
 - [Install on Windows](/doc/install/windows)
 - [Install on Linux](/doc/install/linux)
 
+⚠️ Please try to use the same/latest DVC version on all the team member
+machines. DVC is in active development and compatibility issues may arise
+between significantly different versions.
```

Comment:
What do you think about adding this note @shcheklein @efiop? Per https://groups.google.com/a/iterative.ai/forum/#!msg/support/ZBNM2dydyxU

Reply:
It might be good, but it probably take Ruslan's exact language about backward and forward compatibility and also let's not scary users with `is in active development` phrase :) First of all, only projects are of not any interest are not being actively developed and any mature project keeps in mind compatibility policies. Second, our version number is already enough to scary users :)

Also, it can be a larger topic a bit - we can clarify the policy around the versioning - major, minor, etc. It's important to understand those.

Reply:
let's remove this and create a separate ticket to document this properly?

Reply:
OK, removed. Created 906